src/convert.ts

```diff
@@ -2405,6 +2405,19 @@ export default function convert(config: ConvertConfig): ESTreeNode | null {
         );
       }
 
+      // accessibility, export and static is not allowed in MethodSignature TS1070
```

Comment:
I find this comment confusing, because it doesn't seem like the information is taken into account anywhere. Am I missing something?

Reply:
hmm i can remove this comment, but its more info that those properties should not be present.

if you use them it will throw TS1070 error code. but we are still putting them into ast.

thats a case for alot of nodes, sometimes we pass them as "modifiers" sometimes as properties like this. and if you agree with me i'm going to do PR witch removes them from unsupported properties.